Dockerfile

```diff
@@ -0,0 +1,16 @@
+# When updating check if similar changes need to be made to cmd/otelcontribcol/Dockerfile
+FROM golang:1.14 AS build
+
+WORKDIR /src
+ADD . /src
+
+RUN make otelcontribcol
+
+FROM alpine:latest as certs
+RUN apk --update add ca-certificates
+
+FROM scratch
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY --from=build /src/bin/otelcontribcol_linux_amd64 /otelcontribcol
+ENTRYPOINT ["/otelcontribcol"]
+EXPOSE 55678 55679
```

Comment:
replace 55678 (opencensus) with 55680 (otlp)

Reply:
The runtime package is copied from the other Dockerfile here

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/master/cmd/otelcontribcol/Dockerfile

Should I update both?

Reply:
Ah, yes good idea -- plan is to move from oc to ot